### ISSUE TITLE
Koreader reading statistics sync.

### ISSUE THREAD

**Issue by Zeyadas** [koreader/contrib#8]
I'm sorry but I didn't know where to suggest this.
I was wondering if you can make a plugin to sync the reading statistics on two or more devices, something like the progress sync in koreader.
Thank you.

**Comment by Hzj-jie** [comment]
Yes, I'd like to have it for a while, we may move this to koreader/koreader, as the read statistic is the core function.
